doc/manual/cookbooks/phase_diagram/doc/phase_diagram.tex

```diff
@@ -0,0 +1,83 @@
+To do this, We used the phase inputs implemented in the visco plastic plug-in.
+This serves the goal of visualizing values of reference densities of phases with the real density assigned with a constant value in the model.
+Inputs for this material model are listed here:
+\lstinputlisting[language=prmfile]{cookbooks/phase_diagram/doc/material_model.prm.out}
+Compared to the first example, pyrolitic phases are illustrated on a much finer scale.  % comparison 
+Meanwhile, we can still tell the phase transitions at 410, 520, and 660 km depth, respectively, marked by linear boundaries analogous to a constant Clapeyron slope.
+
+
+\paragraph{Results}
+
+\par Visualization of the model results yields a phase diagram of a pyrolitic mantle (Figure \ref{fig:phase_diagram_ph_density}). % phase diagram
+The field shown here has the reference densities of the pyrolite phases, though settings of phase transitions are over-simplified.
+One may notice that 3 transitions (i.e. one in the olivine system, two in the spinel system) are included for the 660 interfaces, % 3 transitons, higher T
+and they need to be modified at a higher temperature.
+In spite of the complexities of mantle phases, the focus of this first example is to simply illustrate this approach of visualizing it.
+Beyond showing the diagram, We have also used the lineout feature in VISIT to export the data along two vertical lines at T = 1173 and T = 1673 (Figure \ref{fig:phase_diagram_ph_profile}). % linear profile
+The T = 1173 illustrates the buoyancy forces felt by a descending cold slab within the mantle transition zone.
+
+\par Next, we shift to harzburgite by changing the values of the initial compositional field from 0 to 1.  % harzburgite
+For this to occur, following changes are needed:
+\lstinputlisting[language=prmfile]{cookbooks/phase_diagram/doc/harzburgite.prm.out}
+With this change, we can also visualize the phase diagram of harzburgite in Figure \ref{fig:phase_diagram_ph_density}.
+
+\par Moreover, We tested the a pyrolitic lookup table used in the Steinberg material model (Figure \ref{fig:phase_diagram_steinberg_density}).% steinberg model
+The same setup of the initial condition is applied as in the previous case. % initial condition
+The densities, however, are not assigned to the heat capacity anymore.
+Thus the vertical axis would deviate from the axis of pressure a little bit.
+This second setup serves the goal of illustrating a more complex and thus more realistic model of phase transitions.
+Modification for the material model is listed below:
+\lstinputlisting[language=prmfile]{cookbooks/phase_diagram/doc/steinberg.prm.out}  % add 
+
+
+%%%%% figures %%%%%%
+%%%%%%%%% figures1: pyrolite %%%%%%
+\begin{figure}
+\phantom.
+\hfill
+\includegraphics[width=0.9\textwidth]{cookbooks/phase_diagram/doc/pyrolite_harzburgite.png}
+\hfill
+\phantom.
+\caption{\it The field of heat capacity showing values of reference densities for Pyrolitic and Harzburgitic phases.}
```

Comment:
I'm not a geologist, but I think that everywhere else we use lower-case for rock types, right?
```suggestion
\caption{\it The field of heat capacity showing values of reference densities for pyrolitic and harzburgitic phases.}
```